config/manifests/bases/antrea-operator-for-kubernetes.clusterserviceversion.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
     alm-examples: '[]'
     capabilities: Basic Install
     description: An operator which installs Antrea network CNI plugin on the Kubernetes cluster.
-    operators.operatorframework.io/builder: operator-sdk-v1.2.0
+    operators.operatorframework.io/builder: operator-sdk-v0.19.0
```

Comment:
https://github.com/vmware/antrea-operator-for-kubernetes/pull/20#discussion_r582442391 +1

Reply:
The version is updated in commit 3e64796.

Reply:
It seems that 3e64796 didn't update this value in `config/manifests/bases/antrea-operator-for-kubernetes.clusterserviceversion.yaml`? Otherwise we can see a yellow "Outdated" flag on this comment.

Reply:
Thanks, i'll check this.